app/src/main/java/app/revanced/manager/ui/screen/DashboardScreen.kt

```diff
@@ -60,6 +71,8 @@ fun DashboardScreen(
     onAppClick: (InstalledApp) -> Unit
 ) {
     var showImportBundleDialog by rememberSaveable { mutableStateOf(false) }
+    var showBundleTypeSelectorDialog by rememberSaveable { mutableStateOf(false) }
+    var bundleType by rememberSaveable { mutableStateOf(BundleType.Remote) }
```

Comment:
You could remove `showImportBundleDialog` by making `bundleType` nullable and using `?.let`. Just remember to rename it.

Reply:
I don't think that's good for code readability, and the default state of the variable is BundleType.Remote so that would be the preselected bundleType is Remote 
![image](https://github.com/ReVanced/revanced-manager/assets/79272171/97551246-9d9e-499c-b20f-efcb57e35d0d)